src/parser/dash-manifest-parser.ts

```diff
@@ -1,19 +1,18 @@
-// @flow
 import {MpdUtils, ParserUtils, XmlUtils} from './parser-utils';
 import {AdaptationSet} from './adaptation-set';
 import {getLogger} from '@playkit-js/playkit-js';
 
 class DashManifestParser {
-  _logger: any = getLogger('DashManifestParser');
-  _xmlDoc: Document;
-  _adaptationSets: Array<AdaptationSet>;
-  _baseURL: ?string;
+  private _logger: any = getLogger('DashManifestParser');
+  private _xmlDoc!: Document;
+  private _adaptationSets: Array<AdaptationSet>;
+  private _baseURL!: string | null;
 
-  static isValid(): boolean {
-    return window.TextEncoder && window.TextDecoder;
+  public static isValid(): boolean {
+    return Boolean(window.TextEncoder && window.TextDecoder);
```

Comment:
prefer !!(window.TextEncoder && window.TextDecoder);  over Boolean, since its more common and shorter

Reply:
i have to convert it to Boolean otherwise i get TS error (chenged it to the shorter syntax !!(window.TextEnc...))